api-docs.js

```diff
@@ -897,6 +897,210 @@ const apiDoc = {
         },
       },
     },
+    '/wallet/utxo-filter': {
+      get: {
+        summary: 'Return utxos and some helpful information regarding it.',
+        parameters: [
+          {
+            name: 'x-wallet-id',
+            'in': 'header',
+            description: 'Define the key of the corresponding wallet it will be executed the request.',
+            required: true,
+            schema: {
+              type: 'string',
+            },
+          },
+          {
+            name: 'max_utxos',
+            'in': 'query',
+            description: 'Maximum number of utxos to aggregate. Default to MAX_INPUTS (255)',
```

Comment:
`Not exactly 'to aggregate' in the GET. Can be only 'Maximum number of utxos to return. Default to MAX_INPUTS (255)'. What do you think?

Reply:
I think you are absolutely right, thanks!